dali/kernels/audio/mel_scale/mel_filter_bank_gpu.cu

```diff
@@ -122,22 +122,28 @@ class MelFilterBankGpu<T, Dims>::Impl : public MelFilterImplBase<T, Dims> {
     }
   }
 
-  void Setup(ScratchpadEstimator &se, const TensorListShape<Dims> &in_shape) {
+  void Setup(ScratchpadEstimator &se, TensorListShape<> in_shape) {
```

Comment:
```suggestion
  void Setup(ScratchpadEstimator &se, const TensorListShape<> &in_shape) {
```
It looks like you're not modifying it - no need to copy.

Reply:
Leftovers from a previous change I removed finally. Thanks.